packages/headless/src/controllers/smart-snippet/headless-smart-snippet.ts

```diff
@@ -106,9 +197,22 @@ export function buildSmartSnippet(engine: SearchEngine): SmartSnippet {
         question: state.search.response.questionAnswer.question,
         answer: state.search.response.questionAnswer.answerSnippet,
         documentId: state.search.response.questionAnswer.documentId,
+        source: getResult(),
       };
     },
 
+    source: {
```

Comment:
Not really, I added it because originally I made a dedicated controller for `source` and then I chose not to remove it because I was 50/50 on whether we were better off with or without this structural object. I'll move those out of `source`.